source/importing-css-assets-and-javascript/index.html.md.erb

```diff
@@ -176,6 +186,9 @@ You can select and initialise a specific component by using its `data-module` at
   </script>
 ```
 
+When initialised individually, errors are thrown rather than logged.
+You must check your application works without errors or some components will not work correctly.
```

Comment:
Cheers for updating with ~"should"~ "must"

Did you want these lines to flow into each other?

Markdown will need another new line otherwise this one gets removed

```suggestion
When initialised individually, errors are thrown rather than logged.

You must check your application works without errors or some components will not work correctly.
```

```suggestion
When initialised individually, errors are thrown rather than logged. You must check your application works without errors or some components will not work correctly.
```